### PR TITLE
create prism3.2 blog

### PR DESCRIPTION
**Pull request by ksatx**

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 829483813566354eb0af00b25765f7d8267351df

https://deploy-preview-1340--stoplightio.netlify.com

File: netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+In order to support the proxy feature, Prism also needed to provide full validation. While earlier Prism versions could validate input, Prism 3.2 can also validate the **response** data. There is now full callback support for OpenAPI response validation. As mentioned above, you can now use Prism's mock server for complete contract validation. For example:
+
+Say you have an invalid request in your API or are receiving an invalid response that does not match the rules of your OpenAPI specification document. Prism 3.2 will handle both and return an error message.
+
+The mismatches it detects may be subtle, such as a minor schema difference (expected an integer, but a float number gets returned):
+
+```json
+{
+  type: "number",
```
Comment (XVincentX) [inline netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md]:
```suggestion
  "type": "number",
```

File: netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  format: "integer"
```
Comment (XVincentX) [inline netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md]:
```suggestion
  "format": "integer"
```

File: netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+If the data coming from the server only had a name, it would cause a warning.
+
+```json
+{
+ "name": "Clark"
+}
+```
+
+Here is the warning you would see in Prism:
+```[5:55:28 PM] >     [VALIDATOR] ?  error     Violation: response.body should have required property 'surname'```
```
Comment (XVincentX) [inline netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md]:
```suggestion
`[5:55:28 PM] >     [VALIDATOR]. Violation: response.body should have required property 'surname'`
```

File: netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+  robots: 'index, follow'
+  title: >-
+    Open Source Mock Server Gets Smarter
+  image: /images/prism-zetong-li-unsplash.jpg
+  twitter:
+    title: >-
+      Stoplight Blog: Open Source Mock Server Gets Smarter
+    image: /images/prism-zetong-li-unsplash.jpg
+    username: '@stoplightio'
+  og:
```
Comment (paulatulis) [inline netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md]: We don't need this section. Tags can go under Meta as they are above.

File: netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+    title: >-
+      Stoplight Blog: Open Source Mock Server Gets Smarter
+    image: /images/prism-zetong-li-unsplash.jpg
+    description: Stoplight's Prism supports callbacks, proxies, and even more validations.
+    url: https://stoplight.io/blog/prism-open-source-mock-server-gets-smarter
+---
+
+Generated mock servers are an integral part of [API design](https://stoplight.io/design/). API Consumers can interact with your API before you build it, catching issues before you've invested a lot of time writing code.
```
Comment (paulatulis) [inline netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md]:
```suggestion
Generated mock servers are an integral part of [API design](https://stoplight.io/design/). API consumers can interact with your API before you build it, catching issues before you've invested a lot of time writing code.
```

File: netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+```json
+{
+  "type": "number",
+  "format": "integer"
+}
+```
+
+Returned data: 3.2
+
+or the issue could cause a major error, like a missing fields in the expected payload:
```
Comment (paulatulis) [inline netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md]:
```suggestion
or the issue could cause a major error, like a missing field in the expected payload:
```

**Comment by ksatx** [comment]
[Posting Open Source Mock Server Gets Smarter - Vincenzo Byline Blog](https://trello.com/c/mYQGT7sF/351-posting-open-source-mock-server-gets-smarter-vincenzo-byline-blog)

File: netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+ "name": "Clark"
+}
+```
+
+Here is the warning you would see in Prism:
+`[5:55:28 PM] >     [VALIDATOR]. Violation: response.body should have required property 'surname'`
+
+## Support Webhooks in Your Mock Servers
+
+In addition to validating requests and proxy servers, Prism added support for Webhooks, which aren't always easy to test. An API follows instructions when you make a request. Similarly, webhooks or web callbacks fire a request to a URL you specify. The request occurs when a defined state is met, such as a new record added to a system.
```
Comment (paulatulis) [inline netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md]: The word 'webhooks' is capitlized here but not in other use places in this blog. I'm not sure which is more correct, but check for consistency.

File: netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md (hunk 4 above)
Comment (ksatx) [reply netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md]: @paulatulis  Sorry just want to make sure I'm clear on what you're referencing. We don't need the og section at all, so delete lines 37-42, right?

Comment (paulatulis) [reply netlify/blog-posts/prism-open-source-mock-server-gets-smarter.md]: oops sorry that is confusing! This part: 
`og:
      title: >-
           Stoplight Blog: Open Source Mock Server Gets Smarter
    image: /images/prism-zetong-li-unsplash.jpg
    description: Stoplight's Prism supports callbacks, proxies, and even more validations.
    url: https://stoplight.io/blog/prism-open-source-mock-server-gets-smarter`

**Pull request merged**